internal/cmd/config.go

```diff
@@ -151,6 +151,28 @@ func readDiskConfig(gs *state.GlobalState) (Config, error) {
 	return conf, nil
 }
 
+func legacyConfigFilePath(gs *state.GlobalState) string {
+	return filepath.Join(gs.UserOSConfigDir, "loadimpact", "k6", "config.json")
+}
```

Comment:
Why is this a function? It seems it was used only once.

Reply:
Good catch, it is now reused from the other part of the code where it is needed.

Reply:
Nice 🎉